### PR TITLE
TMA-339 Ipdate event fields for session_id, govuk_signin_journey_id and persistant_session_id

### PR DESCRIPTION
**Pull request by jashelb90**
Due to another change to the ADR we need to update the field placement of the following fields:

- session_id
- govuk_signin_journey_id
- persistant_session_id

The fields have now moved to the user sub-object.

Unit tests have been updated to reflect the changes.

### CODE REVIEW

File: event-processing/event-processor/tests/test-events/unknown-audit-event.ts [hunk 1]
```diff
@@ -1,96 +1,96 @@
...
+    extensions: unknown | undefined;
+    new_unknown_field: string;
+}
+
+export interface IAuditEventUserMessage {
+    transaction_id: string;
+    email: string;
+    phone: string;
+    ip_address: string;
+    session_id?: string;
```
Comment (Aidan23Smith) [inline event-processing/event-processor/tests/test-events/unknown-audit-event.ts:22]: Why is this optional here?

Comment (jashelb90) [reply event-processing/event-processor/tests/test-events/unknown-audit-event.ts:22]: I believe they weren't required fields before and that is still the case now? It allows us to not need to provide them when building or test data if we so choose. I.e. in the first unit test we validate the minimum payload. We could arguably remove them now we have put them into the user object.

Comment (Aidan23Smith) [reply event-processing/event-processor/tests/test-events/unknown-audit-event.ts:22]: They are optional, but so are the others here. Why are these the only ones for this file?

**Review by Aidan23Smith — APPROVED** [review]
Looks good to me

**Pull request merged**